### PR TITLE
add multi api docs

### PR DESCRIPTION
**Pull request by zh-hike**
* torch.Tensor.is_cuda

* torch.nn.Softmax2d

* torch.utils.data.WeightedRandomSampler

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## [用法不同]torch.nn.Softmax2d
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md]: 没有这个分类，写仅 paddle参数更多吧

然后下面也加下相关的参数映射

File: docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.utils.data.WeightedRandomSampler.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+```
+paddle.io.WeightedRandomSampler(weights,
+                num_samples,
+                replacement=True)
+```
+
+两者参数完全一致，具体如下：
+
+### 参数差异
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.utils.data.WeightedRandomSampler.md]: 参数映射

File: docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.utils.data.WeightedRandomSampler.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+| PyTorch     | PaddlePaddle | 备注                                                                 |
+| ----------- | ------------ | -------------------------------------------------------------------- |
+| weights     | weights      | 权重序列，需要时 numpy 数组， paddle.Tensor， list 或者 tuple 类型。 |
+| num_samples | num_samples  | 采样样本数。                                                         |
+| replacement | replacement  | 是否采用有放回的采样。                                               |
+| generator   | -            | 采样中的生成器，Paddle 无此参数，可忽略。                            |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.utils.data.WeightedRandomSampler.md]: 标准话术：Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。

Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.utils.data.WeightedRandomSampler.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.utils.data.WeightedRandomSampler.md (hunk 2 above)
Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.utils.data.WeightedRandomSampler.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md (hunk 1 above)
Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+torch.Tensor.is_cuda
+```
+
+### [paddle.Tensor.place](no)
+
+```
+paddle.Tensor.place
+```
+
+两者返回内容不一致，torch 返回的是 bool 型，为了实现此功能，以下为 paddle 版本的替代实现方案
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md]: 这个直接 按组合替代实现 那个分类来吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+## [仅paddle参数更多]torch.nn.Softmax2d
+
+### [torch.nn.Softmax2d](https://pytorch.org/docs/stable/generated/torch.nn.Softmax2d.html?highlight=softmax2d#torch.nn.Softmax2d)
+
+```
+torch.nn.Softmax2d(*args,
+           **kwargs)
+```
+
+### [paddle.nn.Softmax](no)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md]: 文档：https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Softmax_cn.html#softmax

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+## [仅paddle参数更多]torch.nn.Softmax2d
+
+### [torch.nn.Softmax2d](https://pytorch.org/docs/stable/generated/torch.nn.Softmax2d.html?highlight=softmax2d#torch.nn.Softmax2d)
+
+```
+torch.nn.Softmax2d(*args,
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md]: 这个写成 torch.nn.Softmax2d 吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md (hunk 4 above)
Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md]: done

Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md (hunk 5 above)
Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md (hunk 6 above)
Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softmax2d.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+## [组合替代实现] torch.Tensor.is_cuda
+
+### [torch.Tensor.is_cuda](https://pytorch.org/docs/stable/generated/torch.Tensor.is_cuda.html?highlight=is_cuda#torch.Tensor.is_cuda)
+
+```
+torch.Tensor.is_cuda
+```
+
+### [paddle.Tensor.place](no)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md]: 要按照组合替代实现的模板改一下

Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.is_cuda.md]: done

**Review by zhwesky2010 — APPROVED** [review]
LGTM
